.agent/rules/paragraph-account.md

```
@@ -0,0 +1,85 @@
+---
+trigger: model_decision
+description: Account settings for Paragraph
+---
+
+# Manage account
+Source: https://paragraph.com/docs/account/manage-account
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)